app/scripts/controllers/permissions/index.js

```diff
@@ -413,9 +438,6 @@ class PermissionsController {
     await Promise.all(Object.keys(requestedPlugins).map(async pluginName => {
 
       const permissionName = PLUGIN_PREFIX + pluginName
-      result[pluginName] = {
-        permission: permissionName,
```

Comment:
Key renamed `permissionName` and now populated in `processRequestedPlugin`.

Reply:
Ok, I'm fine with this, but in general I lean towards minimizing api-surface changes that don't add behavioral changes, just to minimize context switching for returning readers and module consumers.